### DISCUSSION TITLE
I am starting a new open source project, and would love some feedback

### DISCUSSION THREAD

**Discussion by ljlabs** [collab-community/community#3] [Ideas]
So I am building a tool similar to IFTTT, but focused on web3, with more of a focus on Backend as a service.

It is called crypto flow and it allows you to configure a listener, process the data in some way and write it to some out put.

The POC I have chosen is a workflow that listens to sushi swap, and tweets when a new cryptocurrency is added to the exchange.

For a more detailed explanation I have created a Press Release on [Medium](https://medium.com/@jordaan0/crypto-flow-the-beginning-83a4f06b5cc2) that goes through several of the features, please check it out and let me know what you think

**Comment by KendallDoesCoding** [comment]
I'm not familiar with this type of stuff so can't help you.

**Reply by KendallDoesCoding** [reply]
Sorry

**Reply by ljlabs** [reply]
No problem, was just intrested on some outside opinions 😁

**Comment by Abhinavkar** [comment]
I can help in the web dev part but CRYPTO stuff is way out of my league but I can start learn once you start teach a Lil so all community can learn too. @ljlabs

**Reply by ljlabs** [reply]
That's a great idea, I'd be happy to put something together once I get through the POC crypto flow project I'm working on. 
It may even be interesting to teach crypto from the perspective of that project, as it's designed to help make web3 project development easier 😀

**Reply by ljlabs** [reply]
I have written a trail How-To style post in crypto, and wanted to get opinions. 
https://medium.com/@jordaan0/how-to-read-sushi-with-python-4780371af049

Please note that I am very far from a professional writer, and I have a feeling that I have a lot of progress to make when it comes to writing. But I just want to start, and I will get better as I do more

**Comment by KendallDoesCoding** [comment]
Same!

**Comment by ShubhamJagtap2000** [comment]
I would like to hear what activities your organization does/conducts. I will be willing to join the community as well.
